client/constants/interfaces.ts

```diff
@@ -74,3 +74,7 @@ export interface FundingProjectCostInfoInterface {
   applicantShare: string;
   totalRequest: string;
 }
+
+export interface SupportDocsInterface {
+  docList: any[string],
```

Comment:
I'm not familiar with this type? If we aren't rushing towards a release let's figure out something that doesn't use `any`